webviz_subsurface/plugins/_well_cross_section_fmu.py

```diff
@@ -0,0 +1,580 @@
+                layout["uirevision"] = "keep"
+            fencespec = [[coord[0], coord[1]] for coord in xsect.fence]
+            return {"data": xsect.data, "layout": layout}, fencespec
+
+        @app.callback(
+            [
+                Output(self.ids("map_wrapper"), "style"),
+                Output(self.ids("show_map"), "children"),
+            ],
+            [Input(self.ids("show_map"), "n_clicks")],
+            [State(self.ids("map_wrapper"), "style")],
+        )
+        def _show_map(nclicks, style):
+            """Show/hide map on button click"""
+            btn = "Show Stddev Map"
+            if not nclicks:
+                raise PreventUpdate
+            if nclicks % 2:
+                style["visibility"] = "visible"
+                btn = "Hide Map"
+            else:
+                style["visibility"] = "hidden"
+            return style, btn
+
+        @app.callback(
+            [Output(self.ids("map"), "layers"), Output(self.ids("map"), "uirevision")],
+            [
+                Input(self.ids("fencespec"), "data"),
+                Input(self.ids("stddev-surface"), "value"),
+                Input(self.ids("ensembles"), "value"),
+            ],
+        )
+        def _render_surface(fencespec, surfacename, ensemble):
+            """Update map"""
+            intersect_layer = {
+                "name": "Well",
+                "checked": True,
+                "base_layer": False,
+                "data": [{"type": "polyline", "color": "red", "positions": fencespec,}],
+            }
+
+            surface = get_surface_statistics(
+                self.realizations,
+                ensemble,
+                self.surfacefiles[self.surfacenames.index(surfacename)],
+                self.surfacefolder,
+            )["stddev"]
+
+            surface_layer = make_surface_layer(
+                surface, name=surfacename, hillshading=True
+            )
+            return [surface_layer, intersect_layer], "keep"
+
+    def add_webvizstore(self):
+        store_functions = []
+        for ens in list(self.realizations["ENSEMBLE"].unique()):
+            for surfacefile in self.surfacefiles:
+                store_functions.append(
+                    (
+                        calculate_surface_statistics,
+                        [
+                            {
+                                "realdf": self.realizations,
+                                "ensemble": ens,
+                                "surfacefile": surfacefile,
+                                "surfacefolder": self.surfacefolder,
+                            }
+                        ],
+                    )
+                )
+        if self.segyfiles is not None:
+            for filename in self.segyfiles:
+                store_functions.append((get_path, [{"path": filename}]))
+        for filename in self.wellfiles:
+            store_functions.append((get_path, [{"path": filename}]))
```

Comment:
`self.segyfiles` is never `None` i think (`None` on input is turend into `[]` in `__init__`?).

I.e. this can be simplified to something similar to the [cross section plugin](https://github.com/equinor/webviz-subsurface/blob/dda8f411cb8e6ea2710080611cdbab6cdcf59412/webviz_subsurface/plugins/_surface_with_grid_cross_section.py#L424-L427).